test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -1329,6 +1329,146 @@ public void GaugeSerializationMultipleMetricPoints(string instrumentName, long[]
         }
     }
 
+    [Theory]
+    [InlineData(123.45, true, true)]
+    [InlineData(123.45, true, false)]
+    [InlineData(123.45, false, true)]
+    [InlineData(123.45, false, false)]
+    [InlineData(-123.45, true, true)]
+    [InlineData(-123.45, true, false)]
+    [InlineData(-123.45, false, true)]
+    [InlineData(-123.45, false, false)]
+    public void ExponentialHistogramSerializationSingleMetricPoint(double? doubleValue, bool addPrepopulatedDimensions, bool addAccountAndNamespace)
```

Comment:
We should also test for exemplars with ExponentialHistogram.

Reply:
added